jetty-home/src/main/resources/etc/jetty.conf

```diff
@@ -9,3 +9,4 @@
 # in addition to those found in the start.ini file
 # =======================================================
 jetty-started.xml
+jetty-pid.xml
```

Comment:
This might be better as a optional extra behaviour of `org.eclipse.jetty.util.component.FileNoticeLifeCycleListener` (whose name really should better match the xml) rather than as an entirely separate mechanism.

I'm also thinking that both these xmls (or the combined if we go that way) would be best added via a module now, rather than as a special "hidden" argument added by the `jetty.sh` mechanism.   It would be nice if any dry-run command generated by the start process could correctly create/delete the pid file.

So the `jetty.conf` file could instead have a line like:
```
--module=pid,started
```

Reply:
Perhaps keeping the mechanism separate is better? I'm 50:50.   
If we keep them separate, perhaps rename `FileNoticeLifeCycleListener` to `StartedFileLifeCycleListener`